README.md

```diff
@@ -234,6 +234,12 @@ https://tsheporamantso.github.io/portfolio/
 - Twitter: [@kwakuamoafo](https://twitter.com/kwakuamoafo)
 - LinkedIn: [@lawrenceamoafo](https://www.linkedin.com/in/lawrenceamoafo)
 
+👤 **Amarachi Dimkpa**
+
+- GitHub: [@amazinggacee](https://github.com/Amazinggracee)
+- Twitter: [@amazinggaceu](https://twitter.com/amazinggraceu)
+- LinkedIn: [Amarachi Dimkpa](https://linkedin.com/in/amarachi-dimkpa-070643183)
```

Comment:
- [OPTIONAL] You might need to remove authors that are not part of the project.

Reply:
Noted, Thank you.